### DISCUSSION TITLE
Wing area in non-clean configuration

### DISCUSSION THREAD

**Discussion by TheAviationDoctor** [junzis/openap#39] [Q&A]
Hi @junzis,

_Note: this was previously part two of [another question](https://github.com/junzis/openap/discussions/38), which I decided to split._

I have a question related to the [drag polar](https://github.com/junzis/openap/blob/master/openap/drag.py) script, please.

The drag polar calculations in [drag.py](https://github.com/junzis/openap/blob/master/openap/drag.py) make extensive use of the wing area `S` (as one would expect). Furthermore, as I understand, the values provided in the [aircraft data files](https://github.com/junzis/openap/tree/master/openap/data/aircraft) for `S` are for a clean configuration (i.e., flaps/slats retracted).

Therefore, I assume that when calculating e.g. `delta_cd_gear` a takeoff or landing configuration, I would need to add the flaps/slats area to `S` using the `Sf / S` ratio. Is that a reasonable assumption?

On that note, I found that, for the B737 family, the `flap area` is 29-30% of the wing area ([source](http://www.b737.org.uk/techspecsdetailed.htm)) - mentioning it just in case you would like to consider updating `Sf / S` accordingly for this type.

Thank you in advance, @junzis.

**Comment by junzis** [comment]
So, are you suggesting the equation should be `Sf / (S+Sf)`? 
I think in McCormick's book, `S` represents the reference area, which should be a constant value.
